src/NuGet/Microsoft.Net.Compilers.Toolset/Framework/Metalama.Compiler.Framework.Package.csproj

```diff
@@ -32,7 +32,7 @@
 
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_GetFilesToPackage</TargetsForTfmSpecificContentInPackage>
     <!-- Remove NU5128 once https://github.com/NuGet/Home/issues/8583 is fixed -->
-    <NoWarn>$(NoWarn);NU5100;NU5128</NoWarn>
+    <NoWarn>$(NoWarn);NU5100;NU5128;NU5129</NoWarn>
```

Comment:
Comment on what these warnings mean, why they are suppressed, and when (if ever) this suppression should be removed.